src/cmd/ksh93/sh/nvdisc.c

```diff
@@ -395,11 +411,13 @@ static char*	lookup(Namval_t *np, int type, Sfdouble_t *dp,Namfun_t *handle)
 			nv_setsize(SH_VALNOD,10);
 		}
 		block(bp,type);
-		sh_pushcontext(&sh,&buff,1);
-		jmpval = sigsetjmp(buff.buff,0);
+		sh_pushcontext(&sh, &checkpoint, 1); //SH_JMPCMD);
+		jmpval = sigsetjmp(checkpoint.buff, 0); //1);
```

Comment:
I still don't understand the difference. It would be nice to figure this out somehow.